app/config/config.dist.json

```diff
@@ -180,6 +180,9 @@
 			"membership-enabled": false
 		}
 	},
+	"preset-amounts": {
+		"donation": [ 500, 1500, 2500, 5000, 7500, 10000, 25000, 30000 ]
```

Comment:
We still "have to" use this in the templates to remove the duplication.
* https://github.com/wmde/FundraisingFrontend/blob/master/skins/cat17/templates/Donation_Form.html.twig#L51
* https://github.com/wmde/FundraisingFrontend/blob/master/skins/10h16/templates/Donation_Form_Payment.html.twig#L42
* https://github.com/wmde/FundraisingFrontend/blob/master/skins/10h16/templates/Donation_Form_Personal_Data.html.twig#L34

Of course, it would be _again_ a great time to hear that 10h16 does not need to be supported anymore...